helm/wrongsecrets-ctf-party/values.yaml

```diff
@@ -209,6 +209,8 @@ wrongsecretsCleanup:
   enabled: true
   # -- Specifies when Juice Shop instances will be deleted when unused for that period.
   gracePeriod: 2d
+  # -- Specifies if the clean up job should delete the outdated namespaces or just report them. Set to false to only report outdated namespaces.
+  isDelete: false
   # -- Cron in which the clean up job is run. Defaults to once in an hour. Change this if your grace period if shorter than 1 hour
   cron: "0 * * * *"
```

Comment:
```suggestion
  cron: "0,15,30,45 * * * *" 
```

Reply:
This is great but isn't this the same as "*/15 * * * *"?
If so, I wonder which one is better ?

Reply:
I am choosing "0,15,30,45 * * * *" as it may be more verbose for others